### DISCUSSION TITLE
Access Budibase from iPhone

### DISCUSSION THREAD

**Discussion by ihabmzaky** [Budibase/budibase#8679] [Help]
I have budibase self-hosted on digitalocean.  The apps are accessible from desktop browser, android phones but only accessible via VPN from iphone (any browser).  I initially thought my country has the IP censored from mobiles but noted it is working fine on android.  Also through some troubleshooting on Nginx, noted the requests are received from the iphone but I end up getting connection time out on safari.  Did anyone have this problem?

**Comment by melohagan** [comment]
What OS version is your iPhone?

**Reply by ihabmzaky** [reply]
iOS 16.  But same issue with other iphones I tried with maybe older OS versions.

**Reply by melohagan** [reply]
iOS 16 should work fine. There's some issue below 15.

**Reply by ihabmzaky** [reply]
I just updated to the latest 16.1.1 with no luck!  Would a reverse proxy through a domain solve the issue?  My understanding reverse proxy is like a VPN and through VPN, I have no issues.

**Reply by melohagan** [reply]
There's not a huge amount to go on here. Would you be able to answer the following please:

- did you do the 1 step install from the digital marketplace?
- did you alter the self host config?
- are there any specific error codes?
- do you have any logs?
- could you provide an IP?

**Reply by ihabmzaky** [reply]
IP 209.97.138.245
Installed via docker run -d -t --name=budibase -p 80:80 -v /local/path/data:/data --restart unless-stopped budibase/budibase:latest
port 80 instead of 10000
have no errors
which logs would be helpful?

**Reply by ihabmzaky** [reply]
Btw, I tried it on port 10000, also does not work but not giving me a connection time out.  Not sure but maybe iphones don't accept :port in url.

**Reply by melohagan** [reply]
Here's an example for how to retrieve logs for the single container: https://docs.docker.com/engine/reference/commandline/logs/#examples

Otherwise we have been able to access your server on iOS without any issues.

**Reply by ihabmzaky** [reply]
[Logs.txt](https://github.com/Budibase/budibase/files/10075196/Logs.txt)

got it.  Logs for last 6 minutes attached showing the undefined get/200 when I try from my phone.  The rest was when i tried from my browser

**Reply by ihabmzaky** [reply]
[Logs 30m.txt](https://github.com/Budibase/budibase/files/10075274/Logs.30m.txt)
I thought the logs would be to long but it is not.  This file has a 30min log

**Reply by melohagan** [reply]
Hey @ihabmzaky 

Would you be able to join our discord server? https://discord.gg/vAZS6RxH

I've asked around and it was suggested that it might be an IPv4 / IPv6 issue. If you come onto discord we can try to debug through it.

**Reply by melohagan** [reply]
You could also try switching between mobile data and wifi to test both. And have you tried using the IP address e.g. `209.97.138.245` instead of your domain name.

**Reply by ihabmzaky** [reply]
I do not have a domain connected.  I am using the IP only.  I have tried it through mobile data and wifi with no luck.  I have joined Discord and never used it before.  Not sure what you want me to do next?
